.github/scripts/simple-smoke-test.sh

```diff
@@ -29,6 +30,18 @@ function stop_container() {
     docker stop "${container_name}"
 }
 
+function check_java_version() {
+    local expected_major_version=$1
+    echo "Checking Java version - expected ${expected_major_version}"
+    local actual_major_version
+    actual_major_version=$(docker run --rm "${image}" sh -c 'java -version 2>&1 | head -n 1 | awk -F "\"" "{print \$2}" | awk -F "." "{print \$1}"')
+
```

Comment:
echo `actual_major_version` for debugging purposes

Reply:
It's echoed here: https://github.com/hazelcast/hazelcast-docker/pull/816/files#diff-ca3c00912d3244c223ca46144b4cb9ebb73b370d71128734a230d320d8478468R39

Reply:
But thats if they don't match
I am saying echo regardless

Reply:
ce1db912e6d94ee1e2c2e378574a333b3e894ad4